resources/js/Pages/AnnualSummary.vue

```diff
@@ -181,3 +215,12 @@ function getVacationInfo(day) {
   return day.isVacation ? props.vacations[day.date.toISODate()] : props.pendingVacations[day.date.toISODate()]
 }
 </script>
+
+<style lang="css">
+.cursor-default {
+  cursor: auto;
+}
+.cursor-not-allowed {
+  cursor: not-allowed;
+}
+</style>
```

Comment:
Use tailwind classes instead of own css https://tailwindcss.com/docs/cursor#setting-the-cursor-style